# py3.5.2_issue_test
Minimal example to reproduce the "ImportError" issue
